torch/nativert/executor/Placement.cpp

```diff
@@ -0,0 +1,55 @@
+#include <torch/nativert/executor/Placement.h>
+
+#include <ostream>
+#include <fmt/ostream.h>
+
+namespace torch::nativert {
+
+std::ostream& operator<<(std::ostream& os, const Placement& placement) {
+  std::map<std::string, c10::Device> keys;
+  for (const auto& pair : placement.deviceMap_) {
+    keys.try_emplace(pair.first.str(), pair.first);
+  }
```

Comment:
why the std::map? if you just want sorted iteration, make a vector and sort it.

Reply:
> why the std::map? if you just want sorted iteration, make a vector and sort it.

sure, changing to the following:

```
  std::vector<std::pair<std::string, c10::Device>> sorted_keys;
  for (const auto& pair : placement.deviceMap_) {
    sorted_keys.emplace_back(pair.first.str(), pair.first);
  }
  std::sort(sorted_keys.begin(), sorted_keys.end(),
            [](const auto& a, const auto& b) {
              return a.first < b.first;
            });

  bool first = true;
  for (const auto& pair : sorted_keys) {
...
```